### PR TITLE
Update oauth_for_integrations.md

### PR DESCRIPTION
**Pull request by dd-dominic**
### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/developers/integrations/oauth_for_integrations.md [hunk 1]
```diff
@@ -169,6 +169,8 @@ Error
 
 Until an OAuth client is published, you can only authorize the client from the account it was created in (the partner's sandbox account). This error occurs if you try to authorize the client outside of that account before the client is published.
 
+If you've already published your OAuth client, make sure to use the client ID and client secret you received upon submission. Please note that the client secret is displayed only once. If you've lost your client secret, please contact ecosystems@datadog.com for assistance.
```
Comment (eho1307) [inline content/en/developers/integrations/oauth_for_integrations.md:172]:
```suggestion
If you've already published your OAuth client, make sure to use the client ID and client secret you received upon submission. Please note that the client secret is displayed only once. If you've lost your client secret, contact ecosystems@datadog.com for assistance.
```

Comment (neko-dd) [inline content/en/developers/integrations/oauth_for_integrations.md:172]:
```suggestion
If you've already published your OAuth client, make sure to use the client ID and client secret that you received upon submission. The client secret is displayed only once. If you've lost your client secret, contact [ecosystems@datadog.com][14] for assistance.
```

Comment (neko-dd) [inline content/en/developers/integrations/oauth_for_integrations.md:172]: Please also add the following to the bottom of the file:

```
[14]: mailto: ecosystems@datadog.com
```

**Review by eho1307 — APPROVED** [review]

File: content/en/developers/integrations/oauth_for_integrations.md (hunk 1 above)
Comment (neko-dd) [reply content/en/developers/integrations/oauth_for_integrations.md]: Good catch, thanks :)

**Comment by neko-dd** [comment]
Ah, I see that this is a patch and not a branch. Please re-create this with a branch name like `username/description` so that the tests will run, and then I can approve that PR as well.

**Comment by eho1307** [comment]
Closing this in favor of https://github.com/DataDog/documentation/pull/26448

**Pull request closed without merging**